README.md

```diff
@@ -151,8 +151,9 @@ If you have a website, you can query against the usage statistics of your site:
    var e=document.createElement('a');e.setAttribute('href', 'data:text/plain;charset=utf-8,'+encodeURIComponent(JSON.stringify(JSON.parse(localStorage['usage-data-by-id'])[localStorage['config-primary_usage']])));e.setAttribute('download','stats.json');document.body.appendChild(e);e.click();document.body.removeChild(e);
     ```
 3. Save the data to a file in your project.
-4. Give it to Browserslist by `stats` option
-   or `BROWSERSLIST_STATS` environment variable:
+4. Give it to Browserslist by `stats` option,
+   `BROWSERSLIST_STATS` environment variable
+   or import from `browserslist-stats.json` in project folder:
```

Comment:
I think “save to `browserslist-stats.json`” will be much more clear, than “import” (also feel free to put this way as first way).